core/components/mediamanager/model/mediamanager/classes/files.class.php

```diff
@@ -351,8 +351,10 @@ public function getFileHtml($fileId, $template)
                 $versionArr['created_by'] != 0
             ) {
                 $user = $this->mediaManager->modx->getObject('modUser', ['id' => $versionArr['created_by']]);
-                $profile = $user->getOne('Profile');
-                $versionArr['created_by'] = $profile->get('fullname');
+                if ($user) {
+                    $profile = $user->getOne('Profile');
+                    $versionArr['created_by'] = $profile->get('fullname');
+                }
```

Comment:
Add lexicon for unknown user as fallback

```suggestion
                } else {
                    $versionArr['created_by'] = $this->mediaManager->modx->lexicon('mediamanager.files.file_unknown_user');
                }
```